src/Log/Engine/BaseLog.php

```diff
@@ -81,34 +79,17 @@ public function scopes()
     }
 
     /**
-     * Converts to string the provided data so it can be logged. The context
-     * can optionally be used by log engines to interpolate variables
+     * Formats the message to be logged.
+     *
+     * The context can optionally be used by log engines to interpolate variables
      * or add additional info to the logged message.
      *
-     * @param mixed $data The data to be converted to string and logged.
+     * @param string $message The message to be formatted.
      * @param array $context Additional logging information for the message.
      * @return string
      */
-    protected function _format($data, array $context = []): string
+    protected function _format(string $message, array $context = []): string
```

Comment:
Even though this methods simply returns the message augment now, I kept this method as it would be helpful in implementing this feature on 4.1 todo list.

> Improve log message creation so that context variables are formatted into the message string. This allows usage like $logger->log('info', 'Thing {color}', ['color' => $value])